### DISCUSSION TITLE
Mute Indicator Light

### DISCUSSION THREAD

**Discussion by cloaked2222** [iXab3r/MicSwitch#71] [Ideas]
Please consider adding a feature to control a USB light. The light should turn red when muted, and white when unmuted.

Id suggest using the blink(1) light, which works on PC and MacOS. Here's an example: https://thingm.com/blog/2021/8/27/setup-a-mute-indicator-light-for-zoom-w-hammerspoon-amp-blink1

**Comment by iXab3r** [comment]
Looks cool - never heard about it before, although I probably won't have enough spare time in a nearest month or two to property implement support of blink(1). Will check it later.

**Reply by cloaked2222** [reply]
Cool, thanks for considering it.
